tests/evergreen/test_api.py

```diff
@@ -724,7 +725,7 @@ def test_stream_log(self, mocked_api):
 class TestUserPermissionsApi(object):
     def test_permissions_for_user(self, mocked_api, mocked_api_response):
         expected_url = mocked_api._create_url("/users/test.user/permissions")
-        permissions = [{"type": "project", "permissions": {"proj1": {"test": "test"}}}]
+        permissions = [{"type": "project", "permissions": {"proj1": {"test": 1}}}]
```

Comment:
Minor detail: Adjusted the payload to account for the type-hint on the API method.